### PR TITLE
Bump test_sha in perf tests

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANEGLOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala [hunk 1]
```diff
@@ -43,13 +43,11 @@ final class Runner[T <: ReadWriteService, Extra](
...
       implicit val materializer: Materializer = Materializer(actorSystem)
 
-      // FIXME: https://github.com/digital-asset/daml/issues/5164
-      // This should be made configurable
-      val engineConfig = Engine.DevConfig
-
       // share engine between the kvutils committer backend and the ledger api server
       // this avoids duplicate compilation of packages as well as keeping them in memory twice
-      val sharedEngine = new Engine(engineConfig)
+      // FIXME: https://github.com/digital-asset/daml/issues/5164
```
Comment (cocreature) [inline ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala:48]: Why are you pushing unrelated changes here?

**Comment by garyverhaegen-da** [comment]
Removing automerge label until we sort this out. (Thanks @cocreature for also blocking through review.)
